app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationViewModel.kt

```diff
@@ -48,11 +50,19 @@ class ConversationViewModel(
         }
 
     fun saveDraft(text: String) {
-        repository.saveDraft(threadId, text)
+        GlobalScope.launch(Dispatchers.IO) {
```

Comment:
@hjubb probably need to chat about whether there is a nicer way to have this run async without being tied to the view model scope (as this gets called in `onDestroy` and goes out of scope before the async task even starts)